### ISSUE TITLE
Is LedisDB Production Ready?

### ISSUE THREAD

**Issue by clangley** [ledisdb/ledisdb#171]
I am wondering, is ledisdb production ready? I have a use case where LedisDB is a perfect candidate for my storage needs.

**Comment by siddontang** [comment]
Hi @clangley, I know some people and company have already used it in production, but I am still not sure whether it can be production ready or not.
 
But you can have a try, your feedback is very welcome for its improvement. Thank you :-)

**Comment by pkieltyka** [comment]
its been solid for us for over a year of a 10GB data set (rocksdb), on a service doing 100 to 200 req/sec, of which every request would hit ledisdb and its been a rock.

**Comment by siddontang** [comment]
:-) 

I am very surprised that I have been developing it for more than a year !!!

**Comment by sirvon** [comment]
If I can be a bit nosey can @clangley @pkieltyka tell me why you chose this db over redis?
thanks!

**Comment by pkieltyka** [comment]
long-term data persistence and a data set that can grow > 100GB

**Comment by pkieltyka** [comment]
ledisdb isn't the default choice over redisdb, but if you have those two requirements above.. large data set, where keys are greater than memory, and you want the dataset to be stored in a proper on-disk storage engine, then ledisdb is a much better choice, with still great performance (but not as good as redis since thats all in-memory).

**Comment by linked** [comment]
+1

I found Ledis about 4 days ago -- My redis was exceeding 16GB and crashing a $200 server. I downloaded it at 10am, played with it until 10pm (load testing it up to 50GB dataset *four times*, excellent response speeds for my sparse dataset, no changes to my redis code, 0 errors and only 500mb RAM in use), implemented in production by 4am, monitored health until 8am, and since then I have been sitting by the pool. I know it'll take me 2 months to grow to 50GB and I'm confident I don't have to worry until then.

TLDR: I switched my production machines to Ledis in less than a day, saved $150/mo on hardware, and solved my "redis is crashing all the time" problem. I feel good about it; where can I send a donation?

**Comment by siddontang** [comment]
:-)

Thanks @linked , your feedback is the most motivation for me to improve it.

**Comment by linked** [comment]
I would very much like to donate, to provide even more motivation.

**Comment by nsuper** [comment]
@linked How do you backup and restore your ledisdb? Does #176 resolved?

**Comment by lunny** [comment]
We need some showcase on doc.
